lib/plaid.rb

```diff
@@ -51,12 +51,14 @@ def config
     #          as well.
     #
     # Returns a Hash with keys.to_sym (or nil if hash is nil).
-    def symbolize_hash(hash, values: false)
```

Comment:
values is a keyword argument.  See https://robots.thoughtbot.com/ruby-2-keyword-arguments